src/App.tsx

```diff
@@ -31,6 +32,8 @@ export const App = () => {
         isDesktopSideBarOpen={isDesktopSideBarOpen}
         onMobileCloseClick={() => setIsMobileSideBarOpen(false)}
         onDesktopCloseClick={() => setIsDesktopSideBarOpen(false)}
+        isSearchHistoryOpen={isSearchHistoryOpen}
+        onClickSearchHistory={() => setIsSearchHistoryOpen(true)}
```

Comment:
Currently this state value doesn't do anything, but it will soon be used to open the search modal.